lib/scrabble.rb

```diff
@@ -1,7 +1,21 @@
+require_relative 'word_finder'
+
+# scrabble scoring class
 class Scrabble
+  def initialize
+    @finder = WordFinder.new
+  end
 
   def score(word)
-    1
+    if !word || !@finder.exists?(word)
+      return 0
+    end
+    letters = word.upcase.chars
+    total = 0
```

Comment:
The pattern of using an outside local variable and having each modify it is a sign that there is a better enumerable suited for the task.